ai/poc/model_builder.py

```diff
@@ -0,0 +1,98 @@
+from keras.layers import Input, Dense, Flatten
+from keras.models import Model
+import numpy as np
+import json
+
+json_data = json.load(
+    open('./trainning_data/trainning_data.json'))
+
+data = []
+
+
+def clean_data(data):
+    for match in data:
+        if match[0] == 0 or match[1] == 0:
+            data.remove(match)
+            continue
+        if match[2] == 0 or match[3] == 0:
+            data.remove(match)
+            continue
+        if any(len(x) != 15 for x in match[0]) or any(len(x) != 15 for x in match[1]):
+            data.remove(match)
+            continue
+    return data
+
+
+def match_to_data(match):
+    first_x = []
+    second_x = []
+
+    for unit in match['first_player']['units']:
+        if unit is not None:
+            first_x.append(list(unit['stat'].values()))
+    for unit in match['second_player']['units']:
+        if unit is not None:
+            second_x.append(list(unit['stat'].values()))
+
+    sample = (
+        first_x,
+        second_x,
+        match['first_player']['score'],
+        match['second_player']['score']
+    )
+    return sample
+
+for match in json_data:
+    sample = match_to_data(match)
+    data.append(sample)
```

Comment:
Not inside a function ?
The position here is missleading, either in a function or why other "script" lines

Reply:
I have another branch that clean up the ai folder and I know i changed this file there, I'll take account of this comment on the branch :+1: